### ISSUE TITLE
Change names of YAML tags in the photolyze_nitrate_aerosol section

### ISSUE THREAD

**Issue by yantosca** [geoschem/geos-chem#1968]
### Name and Institution (Required)

Name: Bob Yantosca
Institution: Harvard + GCST

### Confirm you have reviewed the following documentation

- [x] [Contributing guidelines](https://geos-chem.readthedocs.io/en/stable/reference/CONTRIBUTING.html)

### New GEOS-Chem feature or discussion

Based on the discussion in #1966, @viral211 suggests renaming the following YAML tags in `geoschem_config.yml` from:
```yaml
    photolyze_nitrate_aerosol:
      activate: true
      NITs_Jscale_JHNO3: 100.0
      NIT_Jscale_JHNO2: 100.0
      percent_channel_A_HONO: 66.667
      percent_channel_B_NO2: 33.333
````
to:
```yaml
    photolyze_nitrate_aerosol:
      activate: true
      NITs_Jscale: 100.0
      NIT_Jscale: 100.0
      percent_channel_A_HONO: 66.667
      percent_channel_B_NO2: 33.333
```

**Comment by yantosca** [comment]
Also tagging @barronh and @ktravis213

**Comment by yantosca** [comment]
Closed by #1998

**Comment by yantosca** [comment]
We can now close this issue as PR #1998 has now been merged into the 14.4.0 development stream.
